scripts/validate_datetime.py

```diff
@@ -0,0 +1,61 @@
+"""
+This is pure Python implementation of DateTime
+
+For doctests run following command:
+
+python3 validate_datetime.py
+
+"""
+
+
+import datetime
+import doctest
+import os
+
+
+def parseOptions():
```

Comment:
```suggestion
def parse_options():
```
Python variable names assume `snake_case`